gramdictru/Scripts/ResultsBox/index.tsx

```diff
@@ -38,25 +38,32 @@ export class ResultsBox extends React.Component<IResultBoxProps> {
                             overflow: "hidden"
                         }}
                         key={this.props.applicationState.searchedTerm}>
-                        {this.props.applicationState.results.map(resultSet => [
-                            <div className={"results-table" + (this.props.applicationState.isShortResult ? " short-results" : "")}>
-                                {resultSet.map(r => <div className="result-entry">
-                        <div className="lemma" style={{textAlign: this.props.applicationState.searchTerm === "" || this.props.applicationState.searchTerm.startsWith('*') ? 'right' : 'left'}}>{r.lemma}</div>
-                        <div className="symbol" dangerouslySetInnerHTML={{ __html: r.symbol }} />
-                        <div className="grammar" dangerouslySetInnerHTML={{ __html: r.grammar }} />
-                    </div>)}
+                        {this.props.applicationState.results.map((resultSet, index1) => [
+                            <div className={"results-table" + (this.props.applicationState.isShortResult ? " short-results" : "")} key={index1}>
+                                {resultSet.map((r, index2) => <div className="result-entry" key={index2}>
+                                    <div className="lemma" style={{ textAlign: this.props.applicationState.searchTerm === "" || this.props.applicationState.searchTerm.startsWith('*') ? 'right' : 'left' }}>
+                                        <label className='entry-hidden'>__</label>
+                                        {r.lemma}
+                                        <label className='entry-hidden'>__&nbsp;</label>
+                                    </div>
+                                    <div className="symbol" dangerouslySetInnerHTML={{ __html: r.symbol }} />
```

Comment:
Adding hidden elements seems like an easy solution but going forward I was planning to add more copying modes, e.g. "Copy as HTML/Plain text/Markdown". I don't think the rendering code is a good place to implement copy-and-paste.

Reply:
I know, 
You can select data to copy -> click right mouse ->  open menu context with some options like Copy as Html, Copy as Plain text, Copy As Markdown. It will be take more time to do it ( I can do it if you want)
But if you want to use Ctrl +C to copy, i think my solution is easy for customer can use.